### PR TITLE
Unaffected killed

### PR DESCRIPTION
**Pull request by mellowCS**

### CODE REVIEW

**Review by Enkelmann — CHANGES_REQUESTED** [review]
Looks good! Just some small typos to fix.

**Comment by Enkelmann** [comment]
Ghidra complains on my computer:
```
Unable to compile class: PcodeExtractor.java (HeadlessAnalyzer) java.lang.ClassNotFoundException: Unable to compile class: PcodeExtractor.java
```

File: ghidra/p_code_extractor/internal/ParseCspecContent.java [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+			}
+		}
+        catch (FileNotFoundException e) {
+            parseException = e;
+        }
+        catch (IOException e) {
+            parseException = e;
+        }
+        catch (SAXException e) {
+			parseException = e;
```
[the remark this answers is not in the document]
Comment (mellowCS) [reply ghidra/p_code_extractor/internal/ParseCspecContent.java:135]: The indentation seems to be correct in the IDE at least

File: ghidra/p_code_extractor/internal/ParseCspecContent.java [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+     * matching the language id. e.g. id = ARM:LE:32:v8
+     * to analyse the correct language.
+     */
+    public static String parseLdefFile(XmlPullParser parser) {
+        String cspecName = null;
+        parser.start("language_definitions");
+        while(parser.peek().isStart()) {
+            XmlElement languageEnter = parser.peek();
+            if(languageEnter.getAttribute("id").equals(languageId.getIdAsString())) {
+                cspecName = getCompilerName(parser);
```
[the remark this answers is not in the document]
Comment (mellowCS) [reply ghidra/p_code_extractor/internal/ParseCspecContent.java:192]: I added the break. For whatever reason this is not noted as outdated.

**Review by Enkelmann — APPROVED** [review]
Now it also works on my computer. :-)

**Pull request merged**